### DISCUSSION TITLE
Issue tracker and discussions

### DISCUSSION THREAD

**Discussion by pgaskin** [cmus/cmus#1092] [General]
@flyingmutant, can you add some discussion categories:

- Troubleshooting
- Integrations

Also, should we close issues which won't realistically be implemented, are generally out of scope, or aren't actionable (e.g. #923, #948, #886, #427,  #278, #216, #135, #21, #3, #1)?

**Comment by flyingmutant** [comment]
> @flyingmutant, can you add some discussion categories:
> 
> * Troubleshooting
> * Integrations

Done.

> Also, should we close issues which won't realistically be implemented, are generally out of scope, or aren't actionable (e.g. #923, #948, #886, #427, #278, #216, #135, #21, #3, #1)?

I don't mind closing the issues.
